### DISCUSSION TITLE
Failure to renew the Github pro for students

### DISCUSSION THREAD

**Discussion by hw88hw88** [community/community#51587] [GitHub Education] [answered]
I am renewing the Github pro for students status. I have been rejected for 5 times and  have tried uploading official documents from my University, my ISIC student card and a screenshot of my student portal.
How to renew my status? Is it a way to get approved?

**Comment by luolitu** [comment]
I'm in the same boat, no matter which method I use, the verification fails, and I'm struggling with this.

**Comment by samus-aran** [comment]
Hi @hw88hw88  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by hw88hw88** [reply, marked as answer]
Hi,
Actually, the bot only showed the reasons repeatedly even though the rejected applications were valid.

Anyway, **my last attempt was successful**. I used two mobile phones, one for taking the photo and one for showing my virtual ID of ISIC.

========================================
The following is the reasons from the bot: (For your REFERENCE)

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.


The text in the photo you used was unable to be read. If using your device camera, make sure the image is clear and legible. For some languages, you may need to capture the original document beside an English translation.


You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.


Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?


The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.


Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.


Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.


The image you submitted does not appear to match the proof type you selected. If your original document is not in English, please take a photo of the original next to an English translation.

**Reply by samus-aran** [reply]
Hi @hw88hw88 Thanks for responding back! I'm glad you were able to receive a successful submission 🎉 

> Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

Looking through your rejection reason the above is extremely important 👆 Allowing browser location is essential to processing whether students are on campus or remote learning. I believe resolving this helped you progressed.

Enjoy being a verified student to [Global Campus](https://education.github.com/globalcampus/student). 

GitHub Global Campus is a portal from which you can access your GitHub Education benefits and resources, all in one place. On the GitHub Global Campus portal, students can:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋

**Comment by IBRAHIMtaiyeroheemat** [comment]
> I am renewing the Github pro for students status. I have been rejected for 5 times and have tried uploading official documents from my University, my ISIC student card and a screenshot of my student portal. How to renew my status? Is it a way to get approved?

Hi,
Actually, the bot only showed the reasons repeatedly even though the rejected applications were valid.

Anyway, my last attempt was successful. I used two mobile phones, one for taking the photo and one for showing my virtual ID of ISIC.
Please how did you do this

**Comment by sevenc-nanashi** [comment]
Same. I sent the application for over 10 times, also followed the suggestion, but still rejected.

> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
